src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  align-items: center;
+  flex-direction: row-reverse;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin: 0;
+    cursor: pointer;
+  }
+
+  &__star:not(:first-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars__star:hover ~ div,
```

Comment:
you should just reuse your solution from `Stars` task. So it should work with different modifiers (1-5 stars)

Reply:
what exactly I have to use if in task with stars there is 5 raws and here we need only one?